src/python/WMCore/MicroService/Unified/Common.py

```diff
@@ -81,6 +85,19 @@ def phedexInfo(datasets):
             blockNodes[item['name']] = nodes
     return blockNodes
 
+def getRequests(status):
```

Comment:
This is not using what I suggested here: https://github.com/dmwm/WMCore/issues/9142#issuecomment-482565706

Reply:
Alan, this is the part of **Unified black box** logic which I didn't touch yet. I want you first to concentrate on parts which I made for first release, and once it is everything in place I'll move on fixing stuff like that.

Reply:
Ohh, sorry, I thought about something else. I'll fix it.